berserk/types/account.py

```diff
@@ -0,0 +1,117 @@
+"""Aliases for account endpoints."""
+from __future__ import annotations
+
+from datetime import datetime
+
+from typing_extensions import TypedDict
+
+from ..enums import AllPerfType, CountTypes, StreamingService
+
+
+class Perf(TypedDict):
+    """A perf."""
+
+    games: int
+    rating: int
+    rd: int
+    prog: int
+    prov: bool
+
+
+class PlayTime(TypedDict):
+    """Account play time."""
+
+    total: int
+    tv: int
+
+
+class Profile(TypedDict):
+    """Public profile of an account."""
+
+    country: str
+    location: str
+    bio: str
+    firstName: str
+    lastName: str
+    fideRating: int
+    uscfRating: int
+    ecfRating: int
+    links: str
+
+
+class StreamerInfo(TypedDict):
+    """Information about the streamer on a given platform."""
+
+    channel: str
```

Comment:
not used in the code (relying on GitHub search)

Reply:
oopsie